### PR TITLE
Extra tweak for making the forkGroup work on build.sbt

### PR DESCRIPTION
**Pull request by lucamilanesio**
When the fork grouping definition is on build.sbt, extra rules
apply, such as the line taken as separator between settings.
It seems as well that the ';' to terminate the def function is needed
to isolate it from the setting definition itself.

### CODE REVIEW

File: src/sphinx/Detailed-Topics/Testing.rst [hunk 1]
```diff
@@ -201,9 +201,8 @@ available with :key:`testGrouping` key. For example in build.sbt:
       def groupByFirst(tests: Seq[TestDefinition]) =
         tests groupBy (_.name(0)) map {
           case (letter, tests) => new Group(letter.toString, tests, SubProcess(Seq("-Dfirst.letter"+letter)))
-        } toSeq
-
-        testGrouping in Test <<= groupByFirst( (definedTests in Test).value )
+        } toSeq;
```
Comment (jsuereth) [inline src/sphinx/Detailed-Topics/Testing.rst:204]: Hmm, we should probably just not have postFix notation here....

``` scala

(tests gorupBy (_.name(0)) map {
  case (letter, tests) => new Group(letter.toString, tests, SubProcess(Seq("-Dfirst.letter"+letter)))
}).toSeq
// maybe a comment then here to make sure the line breaks...
```

**Comment by lucamilanesio** [comment]
Why don't we make it even simpler then, so the sample would be straightforward:

``` scala
testGrouping in Test := (definedTests in Test).value map {
  case test => new Group(test.name, Seq(test), SubProcess(Seq("-Dtest.name="+test.name)))
} toSeq
```

In this case we just run one JVM per test suite.

**Comment by jsuereth** [comment]
Right, but removing the postfix operator, so:

``` scala
testGrouping in Test := (definedTests in Test).value.map {
  case test => new Group(test.name, Seq(test), SubProcess(Seq("-Dtest.name="+test.name)))
}.toSeq
```

I agree that's much simpler.  I think the previous format was never fully updated for 0.13 syntax.  What you suggest is a lot better, would love to take that in.

**Comment by eed3si9n** [comment]
I follow Coding in Style talk by @jsuereth: infix `map`, but use dot-notation for `toSeq` with parens:

``` scala
testGrouping in Test := ((definedTests in Test).value map {
  case test => new Group(test.name, Seq(test), SubProcess(Seq("-Dtest.name=" + test.name)))
}).toSeq
```

His original commit comment reflects the style, but not the comment above.

**Comment by jsuereth** [comment]
Just wanted to check status of this.  Are we going to fix the style in this PR or should I merge and fix in follow on?

**Comment by lucamilanesio** [comment]
Yes, no problem ... will change the style in order to be merged.

**Comment by eed3si9n** [comment]
closing this due to inactivity.

**Pull request closed without merging**